backend/src/controller/controller.ts

```diff
@@ -70,7 +85,7 @@ export const addNewDoctor = async (req: Request, res: Response) => {
     specialty: req.body.specialty as string,
     location: req.body.location as string,
     about: req.body.about as string,
-    active: req.body.active as boolean,
+    status: req.body.status as boolean,
```

Comment:
I know it's not part of this PR so you don't need to fix it here, but I just wanted to note that casting types isn't very safe. There's no guarantee that `status` is actually `boolean`. Instead you should create something called a [type guard](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#typeof-type-guards) that takes in some unknown input, checks to make sure it's the correct type and if it is, either returns back the value or a boolean.

In this case, you'd want to actually check that `req.body.status` is a boolean rather than casting it to a boolean.